frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -147,7 +147,7 @@ class AccountsSummary extends Component<Props, State> {
         if (!data) {
             return null;
         }
-        const accountsByCoinCodes = (data && !data.chartDataMissing) ? this.groupByCoinCode(data.accounts) : {};
+        const accountsByCoinCodes = data ? this.groupByCoinCode(data.accounts) : {};
```

Comment:
Since there is no coin grouping anyway, why not delete `groupByCoinCode()` and `accountsByCoinCodes` and just use `data.accounts`?